packages/react-components/react-toast/src/components/AriaLive/AriaLive.types.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type AriaLiveSlots = {
+  root: NonNullable<Slot<'div'>>;
+  assertive: NonNullable<Slot<'div'>>;
+  polite: NonNullable<Slot<'div'>>;
+};
+
+export type AriaLivePoliteness = 'polite' | 'assertive';
+
+/**
+ * AriaLive Props
+ */
+export type AriaLiveProps = ComponentProps<Partial<AriaLiveSlots>> & {
+  announceRef: React.RefObject<Announce>;
```

Comment:
```suggestion
  announceRef: React.Ref<Announce>;
```

Is it mandatory to have `RefObject`?